### DISCUSSION TITLE
can we have small definition/overview of topic in "Skip Playground"

### DISCUSSION THREAD

**Discussion by GSK-FS** [skiptools/skip#241] [Ideas]
I seen "Skip Playground" on website with conversion of it. I was wondering that it would be nice if there is a little overview about the selected topic. It will also help junior developers to learn and adopt at the same time.

**Comment by marcprux** [comment]
The [Skip Playground](https://skip.tools/playground/) is just a web page on our site where you can experiment and preview how snippets of Swift will be transpiled into Kotlin. It is mostly useful for developers to see how their Swift on iOS will be turned into the Kotlin that runs on Android. It doesn't do anything unique itself, but is mostly useful to educate developers who are curious about how the transpiler works before they dive in an [install Skip](https://skip.tools/docs/gettingstarted/) to start developing their own apps.

When discussing the playground, we find it important to disclaim that even though the language snippet preview tool is implemented as a web app, Skip itself does not connect to it or use it at all during your local development. Skip is a locally-installed tool, and does not upload your code anywhere.
